Jenkinsfile

```diff
@@ -201,8 +219,9 @@ pipeline {
       when {
         beforeAgent true
         allOf {
+          expression { return env.ONLY_DOCS == "false" }
           anyOf {
-            environment name: 'GIT_BUILD_CAUSE', value: 'pr'
+            changeRequest()
```

Comment:
I'm not sure I understand this change in the context of the goal of the PR. Could you explain? Thanks!

Reply:
Good catch, it's unrelated to this one, GIT_BUILD_CAUSE is an env variable that we do generate on the fly with the gitCheckout, and the `changeRequest` is the builtin step to be used. Then we don't need any specific customisation.